src/main/java/org/folio/rest/impl/LoansAPI.java

```diff
@@ -1,21 +1,18 @@
 package org.folio.rest.impl;
 
-import static io.vertx.core.Future.succeededFuture;
-import static org.folio.rest.impl.Headers.TENANT_HEADER;
-
-import java.lang.invoke.MethodHandles;
-import java.util.ArrayList;
-import java.util.Map;
-import java.util.Objects;
-import java.util.StringJoiner;
-import java.util.function.Function;
-
-import javax.validation.constraints.NotNull;
-import javax.ws.rs.core.Response;
-
+import io.vertx.core.AsyncResult;
```

Comment:
We typically use the eclipse style of import ordering - see https://issues.folio.org/browse/FOLIO-1667

Please update this to respect that ordering to reduce the amount of unnecessary flux between changes.